gips/data/sentinel2/sentinel2.py

```diff
@@ -70,7 +71,8 @@ def tile_lat_lon(cls, tileid):
         Returns (x0, x1, y0, y1), which is
         (west lon, east lon, south lat, north lat) in degrees.
         """
-        e = utils.open_vector(cls.get_setting('tiles'), cls._tile_attribute)[tileid].Extent()
+        atileid = tileid.encode('ascii', 'ignore')
```

Comment:
Yes, either the django ORM membrane or else maybe the redis/RQ membrane.  We got a lotta membranes lately.